assets/source/setup-guide/app/components/UnsupportedCountryNotice/index.js

```diff
@@ -47,19 +50,33 @@ export default function UnsupportedCountryNotice( { countryCode } ) {
 					country: <strong>{ countryName }</strong>,
 					settingsLink: (
 						<Link
+							{ ...documentationLinkProps( {
+								href: '/wp-admin/admin.php?page=wc-settings',
+								eventName: 'get_started_notice_link_click',
+								linkId: 'unsupported-country-link',
+								context: 'pinterest-landing', //todo: Use helper function to get context from path see https://github.com/woocommerce/pinterest-for-woocommerce/pull/295
+							} ) }
```

Comment:
WDYT of simply overwriting the `eventName` given by `documentationLinkProps`?

```suggestion
							{ ...documentationLinkProps( {
								href: '/wp-admin/admin.php?page=wc-settings',
								linkId: 'unsupported-country-link',
								context: 'pinterest-landing', //todo: Use helper function to get context from path see https://github.com/woocommerce/pinterest-for-woocommerce/pull/295
							} ) }
							eventName="get_started_notice_link_click"
```

This way, we:
- won't need any changes to `documentationLinkProps`
- won't have to do any refactorings/renamings https://github.com/woocommerce/pinterest-for-woocommerce/issues/297
- KISS

Reply:
Unfortunately, I tested this solution and it didn't work for me neither in the `PreLaunch` component nor the `UnsupportedCountryNotice`. 

Besides that... I don't think it would be a big advantage since you will need to modify context and the other props anyway... so I don't see any problem sending also the `eventName` there.

Reply:
My bad :man_facepalming:, we use event name in `recordEvent` inside `onClick`, and do not forward it as a prop.

You're right, let's send also `eventName`